app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksAdapter.kt

```diff
@@ -193,17 +193,17 @@ sealed class BookmarkScreenViewHolders(itemView: View) : RecyclerView.ViewHolder
         fun update(bookmark: SavedSite.Bookmark) {
             val twoListItem = binding.root
 
-            twoListItem.setContentDescription(
+            twoListItem.setLeadingIconContentDescription(
                 context.getString(
                     R.string.bookmarkOverflowContentDescription,
                     bookmark.title
                 )
             )
-            twoListItem.setTitle(bookmark.title)
-            twoListItem.setSubtitle(parseDisplayUrl(bookmark.url))
-            loadFavicon(bookmark.url, twoListItem.imageView())
+            twoListItem.setPrimaryText(bookmark.title)
+            twoListItem.setSecondaryText(parseDisplayUrl(bookmark.url))
+            loadFavicon(bookmark.url, twoListItem.leadingIcon())
 
-            twoListItem.setOverflowClickListener { anchor ->
+            twoListItem.setTrailingIconClickListener { anchor ->
```

Comment:
I noticed the bookmarks / favorites don't have leading / trailing icons anymore.
![bookmarks_no_icons](https://user-images.githubusercontent.com/7963079/191966831-a7b0f778-5f80-45ad-a51f-fafad4efe00a.png)

Reply:
its visibility was hidden if the values were not passed through xml. It's fixed now :D